checkov/cloudformation/parser/cfn_yaml.py

```diff
@@ -251,4 +256,15 @@ def load(filename: Path, content_type: ContentType) -> Tuple[DictNode, List[Tupl
 
     file_lines = [(idx + 1, line) for idx, line in enumerate(content.splitlines(keepends=True))]
 
+    if content_type == ContentType.TFPLAN:
+        file_size = len(content)
+        if file_size > MAX_IAC_FILE_SIZE:
+            # large files take too much time, when parsed with `pyyaml`, compared to a normal 'json.loads()'
+            # with start/end line numbers of 0 takes only a few seconds
+            logging.info(
+                f"File {filename} has a size of {file_size} which is bigger than the supported 50mb, "
+                "therefore file lines will default to 0"
```

Comment:
@gruebel Let's add to this message an explanation on how to modify this?

Reply:
yeah sure, was not sure, if I should do it or not 😄